assets/javascripts/discourse/widgets/discourse-reactions-list-emoji.js

```diff
@@ -16,11 +16,25 @@ export default createWidget("discourse-reactions-list-emoji", {
     `discourse-reactions-list-emoji-${attrs.post.id}-${attrs.reaction.id}`,
 
   mouseOver() {
-    if (!window.matchMedia("(hover: none)").matches) {
+    if (this._allowHover) {
       this._setupPopper(".user-list");
 
-      if (!this.attrs.users?.length) {
-        this.callWidgetFunction("getUsers", this.attrs.reaction.id);
+      if (
+        !this.attrs.users?.length &&
+        !this.debounceLoadReactions &&
+        !this.loadingReactions
+      ) {
+        this.loadingReactions = true;
+        this.callWidgetFunction("getUsers", this.attrs.reaction.id)
+          .catch(() => {
+            this.debounceLoadReactions = true;
```

Comment:
Any chance we can use [`#debounce`](https://api.emberjs.com/ember/release/functions/@ember%2Frunloop/debounce) from `@ember/runloop`? 🤔

Reply:
Yes this works! Using that instead :)